src/main/java/org/myrobotlab/service/interfaces/LogPublisher.java

```diff
@@ -0,0 +1,30 @@
+package org.myrobotlab.service.interfaces;
```

Comment:
This file doesn't seem to be obeying 2 spaces no tabs. are you using the proper code formatter?